starter/script.js

```diff
@@ -1 +1,113 @@
+          } else if (newArray[2] === 'delegates') {
+            confirm('You paid off delegates.  Great work.');
+        } else confirm('Are you an idiot?  You did not pay off a single delegate');
+        // --> End of NH's 2nd 3rd Level
+      // --> End of NH's 2nd Level
+      } else confirm('You failed to follow simple instrustions.  Reload the page to start over.');
+  // --> End of NH level <--
+} else if (userPrimaryPick === ('n')) {
+      var blackmailOrPoison = prompt('You have allowed Hilary Clinton to win Iowa and Bernie Sanders to win New Hampshire.  Frank is battling for last place with Martin OMalley, you need to do something BIG.  Blackmail the Attorney General to indict Hilary over her emails or poison Bernies dog so he has to leave the campaign.  (Please input "b" to go with blackmail or "p" to posion.)')
+       console.log(userName + " chose to " + blackmailOrPoison);
+      // --> Start of Nevada 2nd Level <---
+      if (blackmailOrPoison === 'b') {
+        var buyingDelegates = prompt('Success.  Hilary is in prison and now it is down to you and Bernie.  Our budget for Nevada is $5 million.  How much do you want to use buying delegates? [Specify $ amount]')
+        console.log(userName + " chose to spend " + buyingDelegates)
+        // --> Start of Nevada's 1st 3rd Level <--
+        if ((buyingDelegates < 3999999) && (buyingDelegates > 1500000)) {
+          confirm('Good idea.  This was enough to secure a victory without raising suspicion.  Looking forward to seeing what you are made of at Super Tuesday.');
+        } else if (buyingDelegates < 1499000) {
+          confirm('You cheapskate, this was not enough to secure a victory');
+        } else if (buyingDelegates > 4000000) {
+          confirm('Did you really think spending that amount would not raise suspicion?  The FBI is now investigating you.');
+        } else confirm('You failed to follow simple instrustions.  Reload the page to start over.');
+        // --> End of Nevada's 1st 3rd Level <--
+      } else if (blackmailOrPoison === 'p') {
+        var resignOrNot = prompt('Brutal mistake.  Mrs. Sander has bounced back and Bernie is surging off of sympathy points.  Do you want to resign as campaign manager? Please input "y" for Yes and "n" for No.')
+        console.log(userName + " answered " + resignOrNot + " to resigning")
+        // --> Start of Nevada's 2nd 3rd Level <--
+        if (resignOrNot === 'y') {
+          confirm('Good luck with your career... in county government');
+        } else if (resignOrNot === 'n') {
+          alert('You have guts, but the Underwoods now own you')
+        } else null
+        // --> End of Nevada's 2nd 3rd Level <--
+      } else null;
+    // --> End of Nevada 2nd Level <--
+  } else confirm('You failed to follow simple instrustions.  Reload the page to start over.')
+
+/**
+The below "||"" was messing this up.  I wasn't able to figure out why, but but I would like to find out!!
```

Comment:
This is a great question! when using the "||", or the "&&" operators, you still need to include the first part of the operation on both sides. So something like:

if (userPrimaryPick === 'Iowa' || userPrimaryPick === 'iowa')